src/content/api/cli.md

```diff
@@ -139,6 +139,8 @@ The `--env` argument accepts various syntaxes:
 | webpack --env.prod --env.min  | `{ prod: true, min: true }` |
 | webpack --env.prod --env min  | `[{ prod: true }, "min"]`   |
 
+T> See this [environment variables](/guides/environment-variables) guide for more information.
```

Comment:
I know it's not a relevant change that you have made, but I noticed we were missing one more --env usage format! 

For example if you wanted to pass an "array" of info to the same property you can do so like this: 

`--env.foo=hello --env.foo=world`

And inside of the config would return the following for `env`

`{foo: ['hello', 'world']}`

Would you be willing to add this information in this PR?

Reply:
:+1: